PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -452,6 +452,30 @@ def miniAOD_customizeCommon(process):
     process.slimmedMETsPuppi.tXYUncForT01Smear = cms.InputTag("patPFMetT0pcT1SmearTxyPuppi")
     del process.slimmedMETsPuppi.caloMET
 
+    process.load('RecoMET.METPUSubtraction.deepMETProducer_cfi')
+
+
+    addToProcessAndTask('slimmedMETsDeep', process.deepMETProducer.clone(), process, task)
+    addToProcessAndTask('slimmedMETsDeepResp', process.deepMETProducer.clone(), process, task)
+    process.slimmedMETsDeepResp.graph_path = 'RecoMET/METPUSubtraction/data/deepmet/deepmet_resp_v1_2018.pb'
+
+    from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+    phase2_common.toModify(
+        process.slimmedMETsDeep,
+        max_n_pf=cms.uint32(12500),
+        graph_path=cms.string("RecoMET/METPUSubtraction/data/deepmet/deepmet_v1_phase2.pb")
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

```suggestion
        max_n_pf=12500,
        graph_path="RecoMET/METPUSubtraction/data/deepmet/deepmet_v1_phase2.pb"
```

same for the rest of this file

Reply:
Done.